### PR TITLE
Update article: View, edit, and delete cookies

### PR DESCRIPTION
**Pull request by dan-wesley**

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:80]: https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning
"Avoid notes, tips, and important boxes. Readers tend to skip over them. It's better to put that info directly into the article text."

Several alternatives:
h4 with a label/heading describing the point.
**Note:** blah

Comment (dan-wesley) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md:80]: @mikehoffms Got it, thanks

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:8]:
```suggestion
ms.date: 09/08/2022
```

Comment (dan-wesley) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md:8]: @mikehoffms you probably noticed, there are two new fields in the cookies table that are blank because I don't know what/why they are.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:35]: "icon in Activity Bar" - pls check to see if wording & captures should show Focus Mode UI or not

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:41]:
```suggestion
   ![The Cookies pane.](cookies-images/application-storage-cookies-expand-cookies-select-origin.png)
```
moving to dedicated image dirs

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: Nice! Although I can see one problem with doing this now: it will make it harder to merge the incoming changes from the `focus-mode-docs-bash` branch. We have tons of image changes coming in that branch, and we're not quite ready to merge them yet. So, we have to maintain the branch in parallel to main for a while still. Therefore, any changes to the path of images in main risk introducing merge conflicts in the focus-mode-docs-bash.
I would prefer if we didn't change any image paths until we merge the focus mode branch. And once that's done, we can move all images automatically in one go, with a script.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: (This is an "Outdated" comment thread.)  @captainbrosset - it's your call, I suggest keeping the original filenames, for tracking.  @dan-wesley

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: Dan (or Capt.) pls close this "Outdated" thread if feasible.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley pls close this thread is done/feasible.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley Pls close this "Outdated" conversation - been resolved.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley Pls close this Outdated conversation if resolved.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:33]:
```suggestion
1. Open DevTools on the webpage you want to edit.  For example, right-click the page and then select **Inspect**, or press F12.  For additional ways, see [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
```
Provide at least one way, inline, in order to avoid having to leave the page.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:60]:
```suggestion
*  **HttpOnly**.  If `true`, this field indicates that the cookie should only be used over HTTP, and JavaScript modification isn't allowed.  See [HttpOnly cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
```
if ui, use bold instead of backticks.  User-entered text in a ui is bolded. Editing code in a code file is backticks.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:62]:
```suggestion
*  **Secure**.  If `true`, this field indicates that the cookie must be sent to the server only over a secure, HTTPS connection.  See [Secure cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
```
backtick if editing code, bold if using UI

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:68]:
```suggestion
*  **Partition Key**. A _partition key_ is the property or path within a document that can be used to distribute data within a partition. Multiple pieces of data that are spread throughout a document and that have the same partition key value are logically grouped together and stored in the same physical partition.
```
pls tech-review "within a partition".  Clarify "to distribute data"; what's the specific context/objective?  @captainbrosset

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:78]:
```suggestion
![Filtering out any cookies that don't contain the text ID](cookies-images/filter-cookies-name.png)
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:96]:
```suggestion
To delete all cookies, click **Clear All** (![Clear All](cookies-images/clear-all-cookies-icon.png)):
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:98]:
```suggestion
![Clearing all cookies](cookies-images/clear-all-cookies.png)
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:66]:
```suggestion
*  **SameParty**. This attribute provides web developers a means to annotate cookies that are allowed to be set or sent in "same-party, cross-site" contexts. 
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:37]:
```suggestion
   ![The Manifest pane](cookies-images/pick-application-no-manifest.png)
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:41]:
```suggestion
   ![The Cookies pane](cookies-images/open-cookies-select-source.png)
```

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md:33]: Probably delete "For additional ways, see [Open DevTools](/microsoft-edge/devtools-guide-chromium/open)."  Not needed, not the focus.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md:68]: I'm not an expert on this at all. So far, the simplest explanation I've read about this is from this section: https://developer.chrome.com/blog/chips-origin-trial/#what-is-chips

I think the partition key field in DevTools is meant to contain URLs (or at least domains, although I've not been able to test it successfully).
Partitioned cookies allow third-party websites to set cookies on client machines, but in a way that prevents other domains to read them.
The "Partition key" column in DevTools is way for users to see when a cookie is partitioned, and if so, which domain it belongs to.

Based on my understanding I think this part of the docs needs to be changed. But, again, I'm new to this, so can't help much more.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: Yes, I would prefer it if we could keep the same image names and paths if possible. This way when merging the `focus-mode-docs-bash` branch, it will just be a question of choosing which image is best.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/storage/cookies.md:35]:
```suggestion
2. Click the **Application** tab to open the **Application** tool. The **Manifest** pane opens:
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]
Left one small suggestion, and then there's the question around the "partition key" field. Let me know what you think in the earlier comment I posted.
Finally, there's the question about the image paths (left a comment earlier too).

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @dan-wesley (or @captainbrosset )  Please close this "Outdated" conversation as Resolved if this has been resolved in the latest files.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/storage/cookies.md
Comment (dan-wesley) [reply microsoft-edge/devtools-guide-chromium/storage/cookies.md]: @mikehoffms, @captainbrosset 
This is an area that I know nothing about. My initial definition is a copy from https://docs.microsoft.com/en-us/azure/cosmos-db/partitioning-overview.

**Comment by captainbrosset** [comment]
> there's the question around the "partition key" field. Let me know what you think in the earlier comment I posted.

I have no idea what happened, but it seems like the comment referenced above never got added to github. It got lost somewhere.
I can't remember exactly what I said in it, but the gist of it is that I think we need to work on the "partition key" field description a bit more. I'll log another task for us to work on this.

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**
